samples/highcharts/blog/polar-btc/demo.html

```diff
@@ -0,0 +1,15 @@
+<script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/highcharts-more.js"></script>
+<script src="https://code.highcharts.com/stock/modules/exporting.js"></script>
```

Comment:
Sorry for the late nitpicking, but could you keep all modules to be loaded for HC? I mean, remove `stock` from URL